api/v2alpha1/python/setup.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import subprocess
+import sys
+import setuptools
+try:
+  from distutils.spawn import find_executable
+except ImportError:
+  from shutil import which as find_executable
+
+NAME = "kfp-pipeline-spec"
+VERSION = "0.0.1"
```

Comment:
For a proof-of-concept/demo purpose, I use 0.0.1 for now. Will update once finalized package name, etc.

Reply:
changed to 0.0.2 with metadata update (adding license)